internal/vcr/vcr.go

```diff
@@ -187,7 +187,16 @@ func GetAPIKeyForVCR(t *testing.T) string {
 		apiKey = "dummy-api-key-for-replay"
 	}
 
-	return apiKey
+	// Get API URL
+	apiURL = os.Getenv("BRAINTRUST_API_URL")
+	if mode == ModeReplay {
+		// use prod in replay mode (prod is not called, this is just a key for VCR)
+		apiURL = "https://api.braintrust.dev"
+	} else if apiURL == "" {
+		apiURL = "https://api.braintrust.dev"
```

Comment:
I could collapse these branches but then the comment becomes a little ambiguous 🤷